docs/client.appwebsocket.createclonecell.md

```diff
@@ -49,7 +49,7 @@ Specify the cell to clone.
 </tbody></table>
 **Returns:**
 
-Promise&lt;[ClonedCell](./client.clonedcell.md)<!-- -->&gt;
+Promise&lt;import("../admin/types.js").[ClonedCell](./client.clonedcell.md)<!-- -->&gt;

```

Comment:
Same here. Is this expected?

Reply:
https://github.com/holochain/holochain-client-js/pull/337/commits/cfa06d461ca78b51e41b52f08947bd15c8a507db